pkg/web/apps/apps_http_handler.go

```diff
@@ -45,28 +47,64 @@ func (a *httpHandler) GetApps(c echo.Context) error {
 	return c.JSON(http.StatusOK, apps)
 }
 
-// GetAppByID returns an app with the ID in JSON format from the AppService
-// TODO: Implement
 //GetAppById returns a app with the ID in JSON format from the AppService
 func (a *httpHandler) GetAppByID(c echo.Context) error {
-	apps, err := a.Service.GetApps()
+	app, err := a.Service.GetAppByID(c.Param("id"))
 
 	if err != nil {
 		return err
 	}
 
-	return c.JSON(http.StatusOK, apps)
+	return c.JSON(http.StatusOK, app)
 }
 
-// UpdateApp returns an app updated with the ID in JSON format from the AppService
-// TODO: Implement
 //UpdateApp returns a app updated with the ID in JSON format from the AppService
-func (a *httpHandler) UpdateApp(c echo.Context) error {
-	apps, err := a.Service.GetApps()
+func (a *httpHandler) UpdateAppVersions(c echo.Context) error {
```

Comment:
@davidffrench is this implementation in the way that you are looking for? I mean, can we move with the UpdateAppVersions just receiving the versions or would like that it received the app + versions and just update versions?

Reply:
This should just receive versions